src/plugins/plot/MctPlot.vue

```diff
@@ -735,20 +760,24 @@ export default {
             }
         },
 
-        onTickWidthChange(width, fromDifferentObject) {
-            if (fromDifferentObject) {
+        onTickWidthChange(data, fromDifferentObject) {
```

Comment:
We could update the parameters to be `onTickWidthChange({ width, yAxisId }, fromDifferentObject)`

Reply:
I tried that, but got confused while coding, so I changed it :)